azure-pipelines.yml

```diff
@@ -15,4 +15,4 @@ pool:
   vmImage: 'VS2017-Win2016'
 
 steps:
-- template: scripts/step-build-katas.yml
\ No newline at end of file
+- template: scripts/step-build-katas.yml
```

Comment:
Please exclude this file from the changes if it is whitespace-only change. Did you mean to add a step validate-notebooks to it instead?

Reply:
Remove the whitespace change. 
While Andres has put together a script to verify notebooks, I don't think we are quite ready yet to add a suitable build step.